packages/nodejs/src/scope.ts

```diff
@@ -8,7 +8,8 @@
  * Copyright 2018, Google LLC
  */
 
-import { NodeSpan, Func } from "@appsignal/types"
+import { Func } from "@appsignal/types"
```

Comment:
> If we update this usage we can remove all Node.js types from the types package.

I see here that we still import `Func` from it. What's the plan for the types package? Would we want to get rid of this usage as well in a future PR?

Reply:
This `Func` type is shared between the JS and the Node.js integrations. When we removed the usage of Node interfaces from the types package, we didn't remove the shared ones.

I think they're good as they are right now to avoid duplication between the two monorepos.

Reply:
Agreed. I think it's fine to keep these for now.